src/renderer/components/wallet/Settings.tsx

```diff
@@ -148,10 +131,17 @@ const Settings: React.FC<Props> = (props: Props): JSX.Element => {
           <StyledCard>
             <Row>
               <Col span={24}>
-                <StyledPlaceholder>{intl.formatMessage({ id: 'setting.midgard' })}</StyledPlaceholder>
-                <StyledClientLabel>128.128.128.128:8080</StyledClientLabel>
+                <StyledPlaceholder>{network === Network.MAIN ? 'Binance chain' : 'Midgard API'}</StyledPlaceholder>
+
+                <StyledClientLabel>
+                  {pipe(
+                    address,
+                    O.getOrElse(() => intl.formatMessage({ id: 'setting.notconnected' }))
+                  )}
```

Comment:
^ Here the url of Midgard (for `main` and `testnet`) should by displayed (not an `Address`).

Reply:
actually this is an endpoint address (url or ip). Seems address is another one prop :sweat_smile:

Reply:
`Address` in props is an address of a Binance account.